### DISCUSSION TITLE
🌙 Dark theme for the docs

### DISCUSSION THREAD

**Discussion by gy-mate** [sqlalchemy/sqlalchemy#10879] [Ideas]
A dark theme for the [documentation](https://docs.sqlalchemy.org/en/20/index.html) would be really appreciated when opening it during the night.

_I know that the websites are generated by [zeekofile](https://github.com/sqlalchemyorg/zeekofile/) but I don't see feedback forms available there. Or maybe it's just about the Sphinx theme that is used?_

**Comment by CaselIT** [comment]
Hi,

That would be a welcome feature, but likely a very time consuming one.
The docs website is actually generated using https://github.com/sqlalchemyorg/zzzeeksphinx, but the main size it indeed uses that other project.

I'm not sure if it may make sense investigating the use of one of the "newer" sphinx themes, like for example [furo](https://github.com/pradyunsg/furo) or [PyData Sphinx Theme](https://github.com/pydata/pydata-sphinx-theme), to replace the custom one that's currently in use in sqlalchemy.

cc @zzzeek

**Reply by gy-mate** [reply]
Yeah I get it—thank you for the quick reply!

The [PyData Sphinx Theme](https://github.com/pydata/pydata-sphinx-theme) would be really neat though, I think that is the most functional and good-looking one among the newer themes.

**Comment by zzzeek** [comment]
> 
> cc @zzzeek

i dont even know how theme selectors work these days, just a set of CSS tags and some kind of JS adds to....the top level html element?    all of our CSS is local so we dont need to use another sphinx theme, we'd just need someone to go into the site's site.css and then zzzeeksphinx docs.css and do what they want with color overrides.
